src/plugins/formulas/formulas.js

```diff
@@ -32,6 +31,20 @@ Hooks.getSingleton().register('afterSheetRemoved');
 Hooks.getSingleton().register('afterSheetRenamed');
 Hooks.getSingleton().register('afterFormulasValuesUpdate');
 
+/**
+ * Try-catch with an empty catch block as an expression.
+ *
+ * @param {Function} fn Function to surround with try-catch.
+ * @returns {*}
+ */
+const attempt = (fn) => {
+  try {
+    return fn();
+  } catch (_) {
+    // eslint-disable-next-line no-empty
+  }
+};
```

Comment:
Why is this wrapper function necessary?

Reply:
It will suppress error msg from HF that the cellAddress is invalid. Users should know why it does not work, right?

Reply:
Yea I agree it's not pretty, after bumping hf I discovered that we were often calling `hf#getCellType`, `hf#isItPossibleToAdd(Rows|Columns)` and `hf#off` with invalid parameters, mostly wrong indexes. This should be covered by `Rethink renderable/logical/physical indexes support` in https://github.com/handsontable/handsontable/issues/8511.

Reply:
Specifically, hot itself returns `null` when calling `toPhysical(Row|Column)` with an out of bounds index.

Another dirty fix with one less `attempt` call, all the tests pass but there might be edge cases: https://github.com/handsontable/handsontable/pull/8669/commits/29fcdfb47b20decf8004f931f5a414fc378f1b4f

Reply:
When hot returns `null` from `toPhysical*` we should handle that internally and don't call HF.  Same thing when it returns something that is out of bounds of HF maxCols and maxRows. But as long as we keep them HF and HOT in sync on those we can skip it.

"Rethink indexes" does not mean we can use invalid Infinity values now.

Reply:
I removed the `attempt` function in https://github.com/handsontable/handsontable/pull/8669/commits/c52a4f85f0966f7c2eeed8e267a374aff3480695